### DISCUSSION TITLE
Integrating prosemirror-math

### DISCUSSION THREAD

**Discussion by Sadashii** [outline/rich-markdown-editor#615] [Q&A]
I am trying to integrate prosemirror-math plugin to my Editor instance and after going through the previous issues, it was no help, any help appreciated.

**Comment by dominiczy** [comment]
There is a previous discussion on this https://github.com/outline/rich-markdown-editor/discussions/424, but I'm also struggling to intregrate prosemirror-math and katex, help extremely appreciated!
